datalake/backend/tests/utils.py

```diff
@@ -239,6 +244,45 @@ def __exit__(
         self._item.delete()
 
 
+class ProcessingAsset:
+    def __init__(
+        self,
+        asset_id: Optional[str] = None,
+        item_index: Optional[str] = None,
```

Comment:
None of the calls to this method specify `item_index`, so this argument should probably be removed.

Reply:
- [ ]